### PR TITLE
RATEST-321: Platform Testing installation should run github actions

### PR DESCRIPTION
**Pull request by kdaud**
Ticket [ID](https://issues.openmrs.org/browse/RATEST-321)

Currently [TestingInstall](https://github.com/openmrs/openmrs-contrib-qaframework/blob/master/qaframework-bdd-tests/src/test/java/org/openmrs/contrib/qaframework/automation/TestingInstallSteps.java) test is not running actions

### CODE REVIEW

File: qaframework-bdd-tests/src/main/java/org/openmrs/contrib/qaframework/helper/InitialSetupPage.java [hunk 1]
```diff
@@ -152,7 +152,8 @@ public void waitForSetupToComplete() {
 		new WebDriverWait(driver, Duration.ofSeconds(720L))
```
Comment (sherrif10) [inline qaframework-bdd-tests/src/main/java/org/openmrs/contrib/qaframework/helper/InitialSetupPage.java:152]:
```suggestion
		new WebDriverWait(driver, Duration.ofSeconds(720L)).until(ExpectedConditions.or(ExpectedConditions.urlContains("index.htm"),ExpectedConditions.urlContains("home.page")))));
```

Comment (kdaud) [reply qaframework-bdd-tests/src/main/java/org/openmrs/contrib/qaframework/helper/InitialSetupPage.java:152]: By having them on one line it exceeds the recommended line width as per the wiki at https://wiki.openmrs.org/display/docs/Developer+How-To+Setup+And+Use+Eclipse

File: .github/workflows/platform-install-test.yml [hunk 2]
```diff
@@ -0,0 +1,48 @@
+name: Platform Testing Installation
+on:
```
Comment (sherrif10) [inline .github/workflows/platform-install-test.yml:2]: @kdaud This looks good.However one question, is there a big difference with one of platform advanced installation testing ?

Comment (kdaud) [reply .github/workflows/platform-install-test.yml:2]: The difference comes when selecting the type of installation during the set up, their respective feature files shows it clearly.

For Platform Advanced installation

```js
  
Feature: New Advanced Installation

  @selenium
  @initialSetup
  @advancedInstall
  Scenario: Advanced Platform Installation
    Given User selects Language on step one of Advanced Installation
    And User selects Advanced Installation Type
    When User enters all Advanced Installation steps
    Then System should run Advanced installation
```
For Platform Testing Installation

```js
Feature: New Testing Installation

  @selenium
  @initialSetup
  @testingInstall
  Scenario: Testing Platform Installation
    Given User selects Language on step one of Testing Installation
    And User selects Testing Installation Type
    When User enters all Testing Installation steps
    Then System should run Testing installation
```

**Review by sherrif10 — APPROVED** [review]
Thanks @kdaud  cc @dkayiwa

**Comment by sherrif10** [comment]
The changes i shared are not included in the platform installations workflow of which we need them to be like other refapp workflows. We need to add them as well.

**Comment by kdaud** [comment]
The changes in this PR are within the packege _"qaframework-bdd-tests/src/**/*.*"_ hence triggering all the tests for RefApp 2.x to run.

**Comment by kdaud** [comment]
@sherrif10  Is this PR ready for merge?

File: .github/workflows/platform-install-testing.yml [hunk 3]
```diff
@@ -0,0 +1,48 @@
+name: Platform Testing Installation
+on:
+  push:
+    branches: [ master ]
+  repository_dispatch:
+    types: [qa]
+  workflow_dispatch:
```
Comment (sherrif10) [inline .github/workflows/platform-install-testing.yml:7]: Isnt this supposed to checkout the segregated path ?

Comment (kdaud) [reply .github/workflows/platform-install-testing.yml:7]: All the platform UI tests do not run on PRs according to their set up, they only run on a push done on the module. So there it has no value to add segregated path to them.

**Review by sherrif10 — APPROVED** [review]
Thanks

**Pull request merged**